### PR TITLE
Convert UI builds list to table.

### PR DESCRIPTION
**Pull request by nedSaf**
#172

### CODE REVIEW

File: client/app/views/dashboard/builds/builds.html [hunk 1]
```diff
@@ -47,13 +47,25 @@
...
-            </li>
-          </ul>
+        <div class="table-scrollable builds" ng-show="builds.length">
+          <table class="table table-hover">
+            <thead>
+            <tr>
+              <th>Name</th>
+              <th>Repository</th>
+              <th>Created</th>
+              <th>PR Status</th>
```
Comment (amitaibu) [inline client/app/views/dashboard/builds/builds.html]: remove PR

**Comment by nedSaf** [comment]
Update: Builds table.
localhost_9000__code_327ea5776341ee13e1fc__builds

**Comment by nedSaf** [comment]
Update: Added filter
gilter

**Comment by nedSaf** [comment]
Update: Filtering.
filter

**Comment by nedSaf** [comment]
Update: the pusher is working well, When adding a new build manually the entity in restful is without any fields, therefore the entity is shown without the repository.
localhost_9000___builds

**Comment by amitaibu** [comment]
ready for review?

**Comment by nedSaf** [comment]
The filter is not fully tested, ran some tests locally and it worked, I can run full tests tomorrow. 

Nader Safadi
Team lead
Gizra LTD

**Comment by nedSaf** [comment]
@amitaibu PR is ready for code review.

File: shoov/modules/custom/shoov_restful/plugins/restful/node/ui_builds/1.0/ShoovUiBuildsResource.class.php [hunk 2]
```diff
@@ -31,7 +31,7 @@ public function publicFieldsInfo() {
       'resource' => array(
         'repository' => array(
           'name' => 'repositories',
-          'full_view' => FALSE,
+          'full_view' => TRUE,
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_restful/plugins/restful/node/ui_builds/1.0/ShoovUiBuildsResource.class.php:34]: can you please confirm that running UI regression tests still works properly after this change.

Follow [tutorials](http://shoov.io/tutorials/lesson1-install/) if needed

Comment (nedSaf) [inline shoov/modules/custom/shoov_restful/plugins/restful/node/ui_builds/1.0/ShoovUiBuildsResource.class.php:34]: I ran UI regression tests on the `test-example` repo and the tests got pushed into the list as expected, Is this what you mean ?

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**